src/types.ts

```diff
@@ -135,14 +135,25 @@ export interface Page {
   embeddables: EmbeddableWrapper[];
 }
 
+export interface Project {
+  about: string | null;
+  footer: string | null;
+  help: string | null;
+  logo_ap: string | null;
+  logo_lara: string | null;
+  project_key: string;
+  title: string | null;
+  url: string | null;
+}
```

Comment:
I think for most variables, interfaces, etc. in the AP, we tend to use camelCase instead of snake_case.  I would have expected to see `logoAp`, `logoLara`, and `projectKey`.  However, maybe this has drifted or has changed since I last worked on the AP?  I know we've now had a lot of engineers on the project so there might be some mixing and matching of style preferences.

Reply:
These types are for the export structure turned by LARA. That structure has historically used snake_case, so it seems best to stick with it so the format is internally consistent.